pkg/clusteragent/admission/mutate/autoinstrumentation/auto_instrumentation_test.go

```diff
@@ -255,7 +255,7 @@ func TestInjectAutoInstruConfigV2(t *testing.T) {
 			name: "all-lib.config",
 			pod: common.FakePodSpec{
 				Annotations: map[string]string{
-					"admission.datadoghq.com/all-lib.config.v1": `{"version":1,"runtime_metrics_enabled":true,"tracing_rate_limit":50,"tracing_sampling_rate":0.3}`,
+					"admission.datadoghq.com/all-lib.config.v1": `{"version":"1","runtime_metrics_enabled":true,"tracing_rate_limit":50,"tracing_sampling_rate":0.3}`,
```

Comment:
Another note: 

I think the presence of the key in the JSON might refer to `config.v1` and not the service dd_version, and it's possible there's no validation in place to confirm this?